README.md

```diff
@@ -33,6 +35,15 @@
 
 NuoDB Insights is a visual monitor tool that aids NuoDB practitioners in monitoring NuoDB database health, resource consumption, and application workload processed in real-time and historically using an intuitive graphical interface. It can be installed at database startup or after. It also installs locally on the same nodes/hosts your database runs and supports all NuoDB database deployment environments: Kubernetes, Docker, and physical host / Virtual Machine environments. 
 
+## Requirements
+
+| Dependency | Description & version                                 |
+|-----------|--------------------------------------------------------|
+| [NuoDB](https://nuodb.com/) | A distributed SQL database. 4.0 or newer |
+| [NuoDB Collector](https://github.com/nuodb/nuodb-collector)   | The stats collector daemon. 1.1.0 or newer |
+| [InfluxDB](https://portal.influxdata.com/downloads/)    | Time-series database. Only version 1.X is supported. Recommended 1.8 or newer.  |
```

Comment:
Suggest for the last sentence in the Description: 
"NuoDB recommends version 1.8 or later."
or
"Version 1.8 is recommended."

Reply:
will do!